src/Oro/Bundle/DataGridBundle/Resources/config/mass_actions.yml

```diff
@@ -82,7 +82,7 @@ services:
 
     oro_datagrid.extension.mass_action.type.massedit:
         class: Oro\Bundle\DataGridBundle\Extension\MassAction\Actions\Widget\MassEdit\MassEditAction
-        parrent: oro_datagrid.extension.mass_action.type.window
+        parent: oro_datagrid.extension.mass_action.type.window
```

Comment:
This definition is redundant as there is nothing to extend from `oro_datagrid.extension.mass_action.type.window` service it makes sense to delete parent definition at all.

Reply:
Hey @anyt ,
okay, i doesn't looked deep into the code. 
I've removed the `parent` statement and force pushed the change.

Thx Eric

Reply:
Nope, unfortunately not.
But I already discussed this kind of tool with @Baachi

I've found it in the profiler logs
![screenshot 2018-12-11 at 17 18 59](https://user-images.githubusercontent.com/4678698/49814039-1de8f580-fd69-11e8-9e29-0b27094e105f.png)

Reply:
Logs are enough to automate this check, thank you.